### ISSUE TITLE
Mention the SfApexDoc project as an alternative?

### ISSUE THREAD

**Issue by StoppingBuck** [SalesforceFoundation/ApexDoc#78]
Now that ApexDoc is unmaintained, perhaps it would be smart to link to the SfApexDoc project, which AFAICS is still alive and well at https://gitlab.com/StevenWCox/sfapexdoc/-/wikis/home ?

**Comment by sergeiwallace** [comment]
Agreed! I am glad I checked here after learning that this was no longer maintained.
